projects/manifest/src/model.rb

```diff
@@ -8,7 +8,7 @@ class Model < Type
       alias model_manifest relevant_manifest
 
       def attributes_type
-        Attributes.new(root_manifest, [*path, :attributes_type])
+        Attributes.new(root_manifest, [*path, :declaration_data, :attributes_declaration])
```

Comment:
Nice! This will shrink the massive manifest size a bit. One place I'm going to check is if Manifest::Model has an attributes_declaration method that delegates to declaration_data instead of what's removed. That will make me less scared that this will break things.

Reply:
So everything I saw looks like it goes through declaration_data. So as far as I know this won't break anything! yay!